src/pages/dashboard.js

```diff
@@ -1,24 +1,97 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import UserInfo from './components/UserInformation';
-import Navbar from './components/Navbar';
+import {
+  Box,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Text,
+  useColorModeValue,
+  Tooltip,
+  useToast,
+} from '@chakra-ui/react';
+import { ArrowBackIcon, ChevronRightIcon, CopyIcon } from '@chakra-ui/icons';
+import { useRouter } from 'next/router';
+import Sidebar from './components/Sidebar';
 
 const Dashboard = () => {
   const { t } = useTranslation('dashboard');
+  const router = useRouter();
+  const [apiKey, setApiKey] = useState('');
+  const toast = useToast();
+
+  const handleCopyApiKey = () => {
+    setApiKey('example-api-key');
+    navigator.clipboard.writeText(apiKey);
+    toast({
+      title: 'API key copied',
+      description: 'Your API key has been copied to your clipboard.',
+      status: 'success',
+      duration: 5000,
+      isClosable: true,
+    });
+  };
+
+  const handleGoBack = () => {
+    router.back();
+  };
 
   return (
-    <div className="flex flex-col items-center h-screen">
-      <Navbar to="/" />
-      <div
-        className="flex flex-col px-8 mb-6 lg:justify-between xl:flex-row pt-10
-      lg:pt-32 max-w-2xl lg:max-w-6xl h-full text-gray-800 text-lg lg:text-xl w-full"
-      >
-        <div className="max-w-3xl space-y-4 mb-10 text-gray-600">
-          <h1 className="text-3xl text-gray-700">{t('Dashboard')}</h1>
-          <UserInfo />
-        </div>
-      </div>
-    </div>
+    <Box minH="100vh" bg={useColorModeValue('white', 'white')}>
+      <Sidebar />
+      <Box ml={270} pt={5} display="flex">
+        <Text
+          fontSize="lg"
+          fontFamily="monospace"
+          color="#1570FA"
+          fontWeight="bold"
+          marginRight={3}
+          cursor="pointer"
+          onClick={handleGoBack}
+        >
+          <ArrowBackIcon
+            __css={{
+              fontSize: '1rem',
+              marginRight: '5px',
+            }}
+          />
+          {t('Back')}
+        </Text>
+        |
+        <Text fontSize="md" fontFamily="monospace" marginLeft={3}>
+          {t('Dashboard')} <ChevronRightIcon /> {t('App one')}
+        </Text>
+      </Box>
+      <Box ml={270} pt={4} pr={10} display="flex" justifyContent="space-between">
+        <Text fontSize="2xl" fontWeight="bold">
+          Dashboard
+        </Text>
+
+        <InputGroup style={{ maxWidth: '300px', margin: '0 auto', justifyContent: 'right', marginRight: 0 }}>
+          <Input value="00y23804y29b3d9048hend" background="gray.200" padding={6} readOnly />
+          <InputRightElement padding={6} onClick={handleCopyApiKey} cursor="pointer">
+            <Tooltip label="Copy API key">
```

Comment:
this should be translated
```suggestion
            <Tooltip label={t('Copy API key')}>
```